src/main/java/seedu/duke/commands/AddCommand.java

```diff
@@ -0,0 +1,50 @@
+package seedu.duke.commands;
+
+import seedu.duke.DukeException;
+import seedu.duke.component.Capacitor;
+import seedu.duke.component.Component;
+import seedu.duke.component.Inductor;
+import seedu.duke.component.Resistor;
+import seedu.duke.ui.Ui;
+
+public class AddCommand extends Command {
+    private final String config;
+    private final String component;
+    private final double value;
+
+    public AddCommand(String config, String component, double value) {
+        this.config = config;
+        this.component = component;
+        this.value = value;
+    }
+
+    /**
+     * Executes appropriate methods based on the given command.
+     *
+     * @param ui Ui object.
+     */
+    @Override
+    public void execute(Ui ui) throws DukeException {
```

Comment:
Unsure of the purpose of the ui object in this function

Reply:
Will be used when it's actually implemented. The Ui will need to print what happens with execution, something like my iP.

Reply:
Ohh okie alright gotcha